recipes_source/recipes/tuning_guide.py

```diff
@@ -318,6 +319,42 @@ def gelu(x):
 # GPU specific optimizations
 # --------------------------
 
+###############################################################################
+# Enable Tensor cores
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Tensor cores are specialized hardware to compute matrix-matrix multiplication
+# operations which neural network operation can take advantage of.
+#
+# Hardware tensor core operations tend to use a different floating point format
+# which sacrifices precision at expense of speed gains.
+# Prior to pytorch 1.12 this was enabled by default but since this version
+# it must be explicitly set as it can conflict with some operations which do not
+# benefit from Tensor core computations.
+
+## Tensor computation can be enabled "manually" modifying the matrix multiplication precision
+## The default precision is "highest" which will perform the operation according to the dtype
```

Comment:
I'm not sure why it has ##, did you intend it to be rendered as a code block? I think it would make sense for this text to be a regular para? You can check rendering [here](https://docs-preview.pytorch.org/pytorch/tutorials/2912/recipes/recipes/tuning_guide.html#enable-tensor-cores).